cypress/integration/article-view-spec.js

```diff
@@ -16,6 +16,7 @@ const searchPage = new SearchPage()
 
 describe('Article view', () => {
   beforeEach(() => {
+    localStorage.setItem('usage-data-consent', false) // Don't display usage data consent prompt
```

Comment:
let's set this to the command `navigateToHomePage`, similar to `has-onboard-before`, only set it manually when needed

source : https://github.com/wikimedia/wikipedia-kaios/blob/master/cypress/support/commands.js#L53